front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -544,7 +545,7 @@ export function MessageView({
                 busy={loading}
               />
             ) : (
-              <Avatar visual={user?.image} size="sm" />
+              <Avatar visual={readOnly ? null : user?.image} size="sm" />
```

Comment:
should we pass the name as well if we have it?

Reply:
We don't have it on `ChatMessageType`, and on `ChatSessionType` we have only the `userId`, so merging as is 🙏🏻